aws/resource_aws_ami_copy_test.go

```diff
@@ -39,7 +39,7 @@ func TestAccAWSAMICopy_basic(t *testing.T) {
 	})
 }
 
-func TestAccAWSAMICopy_Description(t *testing.T) {
+func TestAccEC2AMICopy_description(t *testing.T) {
```

Comment:
The [current guidance for writing acceptance tests](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/running-and-writing-acceptance-tests.md#per-attribute-acceptance-tests) uses lowercase for "meta" tests like `_basic` and `_disappears`, but uppercase for attribute-based and other tests. Not sure if we should be following the documentation or updating the documentation to include more explicit guidance.

Reply:
While Go conventions [do allow for capitalized words](https://golang.org/pkg/testing/#hdr-Examples) after the underscore, they apply to types and methods on types. Drawing the analogy to what we're doing, it would be reasonable to apply the same logic. However, I do not think we should have capitalized words after underscores for one simple reason: a badly followed convention is worse than no convention at all. Currently, the capitalized word after an underscore is as likely to tell you nothing about the test as something. As a result, a clear, followable convention would be better. I vote for updating the documentation.

Reply:
> Currently, the capitalized word after an underscore is as likely to tell you nothing about the test as something.

Can you please expand on this?

Reply:
Currently, there are around about 1256 `basic` and `disappears` tests, which use the underscore-lowercase "meta" test convention. There are a further 1904 tests that use underscore-lowercase but many do not seem like "meta" tests:

```go
TestAccAWSAPIGatewayUsagePlan_description()
TestAccAWSAPIGatewayMethod_customrequestvalidator()
TestAccAWSAPIGatewayIntegration_integrationType()
TestAccAWSEBSSnapshot_tags()
TestAccAWSEBSVolume_updateSize()
TestAccAWSEIP_instance()
TestAccAWSGlueTrigger_actions_notify()
TestAccAWSInstance_outpost()
```

Reply:
We should enforce whatever standard we choose. In this case, I think we should pick a standard that improves understandability/readability first, and then update whatever documentation and Semgrep rules to enforce it